packages/environment-check/src/output/markdown.ts

```diff
@@ -254,15 +294,18 @@ function writeMessages(writer: MarkdownWriter, messages: ResultMessage[] = []):
  * Converts the envcheck results to markdown report.
  *
  * @param results - envcheck results
+ * @param isDestinationsChecked - has destinations been checked, default is true
  * @returns - markdown report
  */
-export function convertResultsToMarkdown(results: EnvironmentCheckResult): string {
+export function convertResultsToMarkdown(results: EnvironmentCheckResult, isDestinationsChecked = true): string {
```

Comment:
I've added this as I don't want to implement a check for the existence of destinations in the results, as I've seen in support where customers have no destinations in the results due to configuration issues. 
Therefore I think it might be better to pass the param from app-mod when there has actually been no checked performed.

Reply:
Good consideration, we need a way to differentiate between:
- 0 destinations due to configuration issues
- 0 destinations because we just do an environment check

We could even extend this concept a bit, instead of manually passing this into `convertResultsToMarkdown()` function, could we add a property on the result state that indicates which check were requested? 

E.g. extend type  `EnvironmentCheckResult`
https://github.com/SAP/open-ux-tools/blob/e026fc2f82327df8068c7c4e29f874bf05528eab/packages/environment-check/src/types.ts#L65-L70
with

```typescript
    requestedChecks: Set<Check>
}

type Check = 'environment' | 'destination' | 'destinationResults';

```
We also allow to skip `destinationResults` and the same concept could be applied there. 
That way it would be clear from the results which check was executed and the markdown writer could skip results for not executed checks.

What do you think?

Reply:
Implemented this, I think it will keep the markdown clean when the check has not been performed. Thanks @Klaus-Keller